.theme/modules/content-watch.ts

```diff
@@ -0,0 +1,35 @@
+import { spawn } from "child_process";
```

Comment:
name is a bit too generic, rename this as `prebuild.js`, `autoPrebuild.js` or something clear enough.